src/Header/MainManager.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef Group4_Header_MainManager_H_
+#define Group4_Header_MainManager_H_
+
+#include <netinet/in.h>
+
+#include <vector>
+#include <unordered_map>
+
+#include "HttpListener.h"
+#include "RequestHandler.h"
+
+
```

Comment:
Please remove extra blank lines here. You can use clang-format to do it.

Reply:
Done